src/Interpreters/ServerAsynchronousMetrics.cpp

```diff
@@ -488,6 +524,8 @@ void ServerAsynchronousMetrics::updateHeavyMetricsIfNeeded(TimePoint current_tim
 
         /// Test shows that listing 100000 entries consuming around 0.15 sec.
         updateDetachedPartsStats();
+        /// TODO: Test how heavy this operation is
+        updateMutationStats();
```

Comment:
do you think it makes sense to combine it with `updateDetachedPartsStats()`?

Reply:
Yeah, I think I can try to do this part of `updateDetachedPartsStats` and then maybe rename it to something more generic like `updatePartsStats` or similar.